wallet/db.c

```diff
@@ -458,6 +458,10 @@ static struct migration dbmigrations[] = {
     /* option_static_remotekey is nailed at creation time. */
     {SQL("ALTER TABLE channels ADD COLUMN option_static_remotekey"
 	 " DEFAULT FALSE;"), NULL },
+    {SQL("ALTER TABLE vars ADD COLUMN intval INTEGER"), NULL},
```

Comment:
New table entirely perhaps (`variables`?), with columns "bip32_max_index INTEGER, last_processed_block INTEGER, next_pay_index INTEGER, genesishash BLOB"?

Reply:
No, that's transposing the table contents, and usually results in huge tables with a single row, that is horrible to load due to its size. Might not be a big deal now, but depending on the DB an update results in a rewrite of the entire row.

I strongly prefer the multiple-rows, typed-columns, approach since we have very few fundamental types (`int`, `u64`, `blob`) that we are storing here.